### PR TITLE
Adding Heidel Time parser

### PR DESCRIPTION
**Pull request by aditya-nambiar**

### CODE REVIEW

**Comment by jimkont** [comment]
Do we need all these resource files?
Isn't there a way to get them from maven?

**Comment by aditya-nambiar** [comment]
No the manual ( http://standalone-manual.heideltime.googlecode.com/hg/Manual.pdf  page 4 ) , asks to provide it. And I checked a few implementations too , only the heidel code is through maven the treetagger from what I see needs to be directly provided.

**Comment by jimkont** [comment]
You are right, I didn't look into that detail...
The [TreeTager license](http://www.cis.uni-muenchen.de/~schmid/tools/TreeTagger/Tagger-Licence) though is quite restrictive and vague. DBpedia is mostly research but we want DBpedia to can be commercially exploited.

I will have to check this in more detail...
Good job btw ;)

File: core/src/main/scala/org/dbpedia/extraction/dataparser/DateTimeParser.scala [hunk 1]
```diff
@@ -74,29 +75,201 @@ class DateTimeParser ( context : {
...
+                    return false
+                }
+            }
+        }
+        return true
+    }
+
+    // Converts the output of Heidel Parser which is in TimeX3 format into Date
+    // for eg 2007-04-21 , XXXX-4-2
+    def convTimeMLtoDate(text: String, datatype: Datatype) : Option[Date] =
```
Comment (jimkont) [inline core/src/main/scala/org/dbpedia/extraction/dataparser/DateTimeParser.scala]: you should have a look on how you can process xml from scala ;)
http://alvinalexander.com/scala/xml-parsing-xpath-extract-xml-tag-attributes

**Comment by jimkont** [comment]
At the beginning of p9 in the manual I see there is an option to use the Stanford POS tagger in CLI and the doc says they have a [Stanford wrapper](https://github.com/HeidelTime/heideltime/blob/master/doc/readme.txt#L123) isn't there a way to use the Stanford POS from the code?
e.g. as an option [here](https://github.com/dbpedia/extraction-framework/pull/447/files#diff-c2ce49a664b0818abce3fa13ed80b10bR85)

**Comment by jimkont** [comment]
Let's see what the Heidel meintainers say about this: https://github.com/HeidelTime/heideltime/issues/38

File: core/src/main/scala/org/dbpedia/extraction/dataparser/DateTimeParser.scala (hunk 1 above)
Comment (aditya-nambiar) [inline core/src/main/scala/org/dbpedia/extraction/dataparser/DateTimeParser.scala]: Oh yeah I can probably use it to parse TimeX3 format too. I shall have a look into it.

**Comment by JannikStroetgen** [comment]
I just replied here: https://github.com/HeidelTime/heideltime/issues/38

Please let me know if you have any questions regarding HeidelTime.

Cheers,
Jannik

**Pull request closed without merging**